src/api/spaces/index.ts

```diff
@@ -19,4 +19,15 @@ export class SpacesAPI extends Request {
       params: params
     });
   }
+
+  /**
+   * Get spaces list.
```

Comment:
nit:

```suggestion
   * List spaces.
```

Reply:
I like starting with a verb to show it is an action